src/vizier/services/metadata/metadata_server.go

```diff
@@ -36,6 +36,7 @@ import (
 	"go.etcd.io/etcd/client/pkg/v3/transport"
 	clientv3 "go.etcd.io/etcd/client/v3"
 	"google.golang.org/grpc"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Looks like you need to run `gazelle` to update the build files to add this dep.
If you have `bazel` and the golang toolchain setup `make gazelle` at top of tree should do the trick.